pool_configs/examples/zcl.json

```diff
@@ -28,6 +28,12 @@
         "ca": ""
     },
 
+    "explorer": {
+        "txURL": "https://explorer.yourcoin.com/tx/",
+        "blockURL": "https://explorer.yourcoin.com/block/",
```

Comment:
Sigh, ZCL... I would just leave these empty -- they should default to the else, I believe.